services/api/handlers/users.js

```diff
@@ -1,137 +1,142 @@
 var boom = require('boom');
 
-module.exports = function(request, reply) {
-  if (request.method === 'get') {
-    return request.server.methods.db.findUser(
-      [request.params.user],
-      function(err, result) {
-        if ( err ) {
-          return reply(err);
-        }
+exports.get = function(request, reply) {
+  request.server.methods.db.findUser(
```

Comment:
Sure! Take a look at the API here: http://hapijs.com/api#route-prerequisites

It's small, but I think you can reduce the repeated bit at the top in each of these handlers where you look up the user